features/step_definitions/user_steps.rb

```diff
@@ -40,7 +40,9 @@ def sign_up
   fill_in 'user[family_name]', with: @visitor[:family_name]
   fill_in 'user[email]', with: @visitor[:email]
   fill_in 'user[dob]', with: @visitor[:dob]
+  expect(page).to have_content('Su Mo Tu We Th Fr Sa')
```

Comment:
check that the datepicker popped-up

Reply:
`Given('I know nothing about Cucumber')` 😉 This seems like a bit of a hack... feels like it would break if we run tests in a different locale. Do we have doubts that the date picker would fail to show or close? Shouldn't the test be about choosing a date through it and check that the date is stored in the field and correctly passed along?

Reply:
You're right. Let me see if I can easily make it click through the text in the date picker. Otherwise, I'll just merge this.

Reply:
>Do we have doubts that the date picker would fail to show or close?

It stopped showing and closing because of [this](https://github.com/Australian-Genomics/CTRL/pull/89/files#diff-f5771c91175d8f1848ffc7767983a7778fc66b581af994a49d49a0cf8f249c8bR11) change I made in #89

Reply:
That is generally a good idea! 😉